### ISSUE TITLE
Example on Usage should be updated.

### ISSUE THREAD

**Issue by Beomi** [moaazsidat/react-native-qrcode-scanner#4]
I've got 
```
undefined is not an object(evaluating 'NativeModule.CameraManager.Aspect')
```
Error on launching this, 
but I've fixed this issue with this code.

```
npm install -g rnpm
rnpm link react-native-camera
rnpm link react-native-qrcode-scanner
```

However, there was Security Crash issue with iOS10, which enforces to add Message to info.plist(can  be found at Xcode's `project-info-Privacy - Camera Usage Description`.
I have to add String words and then passed iOS's security crack.

Error message was:
```
The app’s Info.plist must contain an NSCameraUsageDescription key with a string value explaining to the user how the app uses this data.
```

I hope someone can get solution on this.
Here's some info for this:
http://useyourloaf.com/blog/privacy-settings-in-ios-10/

**Comment by moaazsidat** [comment]
@Beomi Thanks for filing this, can confirm that this is an issue, I'll investigate more this week, and see if we can add a fix for this. Feel free to send a PR with the updates to the relevant README changes if you have a bit.

**Comment by moaazsidat** [comment]
Looked into this further, and updated the README with relevant changes. Published in version [0.0.7](https://www.npmjs.com/package/react-native-qrcode-scanner)
